core/api/src/domain/ledger/index.types.d.ts

```diff
@@ -2,6 +2,8 @@ type LedgerError = import("./errors").LedgerError
 type FeeDifferenceError = import("./errors").FeeDifferenceError
 type LedgerServiceError = import("./errors").LedgerServiceError
 
+type PaginationArgs = import("graphql-relay").ConnectionArguments
```

Comment:
Domain should not depend on external libraries.

Reply:
This was already present before the PR. I think it might be showing up as a diff because I had moved it and then added it back and now there might be some whitespace change. I will have a follow up pr that fixes Transaction pagination by using the new types and removing  pending transactions from it. The follow up pr can not be merged till the mobile app has migrated to using the pendingTransactions endpoint.